cordova-lib/src/plugman/fetch.js

```diff
@@ -330,7 +330,7 @@ function copyPlugin(pinfo, plugins_dir, link) {
 
     shell.rm('-rf', dest);
 
-    if(!link && dest.indexOf(path.resolve(plugin_dir)) === 0) {
+    if(!link && dest.indexOf(path.resolve(plugin_dir)+'\\') === 0) {
```

Comment:
Thank you pointed out the problem!
Yes, it should be invalid value for macs/unix.

I should change it to node official separator expression.